situp-benchmarks/mapdb-benchmarks/src/jmh/java/com/amazon/situp/benchmarks/processor/state/MapDbProcessorStateBenchmarks.java

```diff
@@ -0,0 +1,60 @@
+package com.amazon.situp.benchmarks.processor.state;
+
+import com.amazon.situp.plugins.processor.state.MapDbProcessorState;
+import com.google.common.primitives.SignedBytes;
+import org.openjdk.jmh.annotations.*;
+
+import java.io.File;
+import java.util.*;
+
+@State(Scope.Benchmark)
+public class MapDbProcessorStateBenchmarks {
+    private static final int BATCH_SIZE = 100;
+    private static final int NUM_BATCHES = 10000;
+    private static final Random RANDOM = new Random();
+
+    private MapDbProcessorState<String> mapDbProcessorState;
+    private List<Map<byte[], String>> data = new ArrayList<Map<byte[], String>>(){{
```

Comment:
* I was thinking wouldn't be nice to use the data objects we are using in the actual service-map? 
* I was also wishing to benchmark the entire plugin, is this the next PR?

Reply:
Yep, the full service-map plugin benchmark PR is next